src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/model/IndividualRiskScores.kt

```diff
@@ -32,4 +32,13 @@ data class IndividualRiskScores(
 
   @Schema(example = "LOW", description = "")
   @get:JsonProperty("sara") val sara: String? = null,
+
+  @Schema(example = "LOW")
+  @get:JsonProperty("saraRiskOfViolenceTowardsPartner") val saraRiskOfViolenceTowardsPartner: String? = null,
```

Comment:
As discussed, enums with weights of 0,10..50 to do the comparison to return the sara result.